src/api/model_operation.i

```diff
@@ -188,6 +189,50 @@ Tensor GpuPoolingForward(const CudnnPoolingHandle &cph, const Tensor &x);
 
 Tensor GpuPoolingBackward(const CudnnPoolingHandle &cph, const Tensor &dy, const Tensor& x, const Tensor& y);
 
+
+class CudnnRNNHandle {
+ public:
+   CudnnRNNHandle(const std::vector<Tensor> &x,
+       const int feature_size,
+       const int hidden_size,
+       const int mode = 0,
+       const int num_layers = 1,
+       const int  bias = 1,
+       const float dropout = 0.0f,
+       const int bidirectional = 0);
```

Comment:
num_layers, dropout and bidirectional can be exposed to python?

Reply:
this is rectified